7_drivers_munge/src/GCM_driver_utils.R

```diff
@@ -217,10 +217,17 @@ adjust_lake_cell_tile_xwalk <- function(spatial_xwalk, lake_centroids, grid_cell
     st_as_sfc() %>%
     st_as_sf()
 
+  # pull out Lake of the Woods. It falls just outside of the bounding box of returned data,
+  # but has lots of data, so we want to keep it in our lake subset
+  lotw <- filter(lake_centroids, site_id=='nhdhr_123319728')
```

Comment:
hehe this made me chuckle because it is so close to "LOTR" for Lord of the Rings 😄

Reply:
👋 DWALLIN

Reply:
Haha I kept thinking the same thing when I was coding it!